doc/release-notes.md

```diff
@@ -36,49 +36,429 @@ reindex or re-sync the whole chain.
 
 # Notable changes
 
-## CoinJoin improvements
+## ChainLock-based random beacon
+
+To improve long-living masternode quorum (LLMQ) selection, the `v20` hard fork
+will enable a new random beacon using ChainLock signatures as described in
+[DIP-0029](https://github.com/dashpay/dips/blob/master/dip-0029.md) (__TODO__: NOT MERGED).
+ChainLock signatures will be stored in the coinbase transaction to be used as a
+source of randomness during quorum member calculations. This change increases
+the decentralization in the source of randomness and prevents masternodes or
+miners from being able to manipulate the LLMQ process.
+
+## Masternode reward reallocation
+
+Since Evolution masternodes (evonodes) have four times more collateral than
+regular masternodes, they are paid in four consecutive blocks each time they
+are selected for payment. This is to maintain the same incentive structure that
+regular masternodes have. Once the `mn_rr` hard fork activates, 37.5% of the
+masternode block subsidy will be moved into the Platform credit pool and used
+to pay evonodes. Both regular masternodes and evonodes will then receive a
+single reward payment on the Core chain per payment cycle. Evonodes will
+receive the remainder of their block rewards via Platform. Additionally,
+evonodes will receive 100% of the fees generated from Dash Platform since they
+are the only nodes that host it.
+
+## Treasury expansion
+
+Currently, 10% of block rewards are set aside for the Dash DAO treasury which
+funds development and other network efforts. Once the `v20` hard fork goes into
+effect on mainnet, the treasury system allotment will increase to 20% of block
+rewards to align with the proposal approved in September. Miner and masternode
+rewards will change to 20% and 60% respectively upon activation of the change,
+previous reallocation periods are ignored.
+
+## Enhanced hard fork
```

Comment:
No longer v20?

Reply:
it's fully implemented and activated via `v20` afaik